src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -382,9 +380,48 @@ public void validateJudgementsHasJurisdiction(CaseData caseData, boolean partOfM
         return errors;
     }
 
+    public void validateHearingStatusForCaseCloseEvent(CaseData caseData, List<String> errors) {
+        for (var currentHearingTypeItem : caseData.getHearingCollection()) {
```

Comment:
should check to see if collection exists? would we not have a null pointer here if they tried to close the case without having any hearings?

Reply:
sorted